### ISSUE TITLE
[Suggestion] A few ideas what Thaumcraft could do with the Elytra

### ISSUE THREAD

**Issue by Morinor** [Azanor/thaumcraft-suggestions#298]
-A self-air focus couls act as a firework replacement
-(In case we get the Thaumostatic Harness back (I hope so, creativemode-like flight is super useful for building)) 
A combination of elytren and the Thaumostatic Harness (idealy as a body bauble so it doesnt take up armor slots): you can fly with it like with a normal elytra, but it has a toggleable boost that consumes fuel (maybe not vis, bc that would mean you could fly forever)

**Comment by praecipitator** [comment]
Do you mean the Elytra?
No idea in what language your minecraft is, but in the english version, it's just "elytra".

Yes, turning it into a bauble would be good. Body bauble, of course, that makes the most sense. 
I think the Thaumostatic Harness should be a separate item, though. It fulfills a different purpose, namely floating in place (for easier building, for example), instead of getting around fast.

They could both run on potentia, because the TH used to run on that before. Or maybe volatus, because flying.

**Comment by Morinor** [comment]
The plural is elytren, afiak?
I dont think this bauble should replace the normal harness, as you said its really useful for building.
But i think the Thaumostatic Harness would be the best item to link to an advanced elytra.

It could run on essentia, be it volatus or potentia, or maybe it could even run on any fuel source?
Id imagine it would be cool if you could even use coal or sticks, but those would be less efficient and would produce a lot of flux, indicated by a dark trail of black and purple smoke you leave behind when using a bad fuel, and using essentia or alumentum would result in a clean flame that only produces little flux.

**Comment by praecipitator** [comment]
No, "elytra" is a plural already. The singular is "elytron" and would be one of the two wings.

I don't see the point of the powered elytra producing actual flux, aka, poisoning the aura. You would only spread it around in a large area where it can dissipate easily. 
Instead, it should poison anyone nearby, especially the wearer, when running on "dirty" fuel.

You could be able to upgrade it. Elytra Emissions Control! Add a catalytic taint converter :D

**Comment by Morinor** [comment]
Haha, seems like I have to change the title then^^ For some reason I was pretty sure about that one. Whatever :P
Yea, flux was just the first thing that came to mind (Its TC's only form pollution)
The poisoning sounds more effective, now that I think about it.
(The normal Harness could still produce flux tho, if it would use the same fuel mechanic)
